src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int randomIndex = new Random().nextInt(Colors.values().length);
```

Comment:
Random is your dependency here, and also an instance of this class is created each time the method is called. So, I'd recommend initializing 'random' in the class field